### PR TITLE
test(supermarket quality) Updating cookbook to meet Chef Supermarket quality

### PR DESCRIPTION
**Pull request by scottford-lw**
This update is to address Chef Supermarket quality metrics. 

add `CONTRIBUTING.md`
add `TESTING.md`
add `.delivery/project.toml` 
update `.circleci/config.yaml` to exec `delivery local all`

bump cookbook version to `0.2.1`

Closes [#10](https://github.com/lacework/chef-lacework/issues/10)
Closes [#12 ](https://github.com/lacework/chef-lacework/issues/12)

Signed-off-by: Scott Ford <scott.ford@lacework.net>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,8 +10,5 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Run cookstyle
-          command: cookstyle .
-      - run:
-          name: Run rspec
-          command: rspec .
+          name: Run delivery
+          command: delivery local all
```
Comment (afiune) [inline .circleci/config.yml:14]: missing extra line

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+### Code Review Process
+
+Code review takes place in GitHub pull requests. See [this article](https://help.github.com/articles/about-pull-requests/) if you're not familiar with GitHub Pull Requests.
+
+Once you open a pull request, cookbook maintainers will review your code using the built-in code review process in Github PRs. The process at this point is as follows:
+
+1. A cookbook maintainer will review your code and merge it if no changes are necessary. Your change will be merged into the cookbooks's `master` branch and will be noted in the cookbook's `CHANGELOG.md` at the time of release.
+2. If a maintainer has feedback or questions on your changes they they will set `request changes` in the review and provide an explanation.
+
+### Developer Certification of Origin (DCO)
```
Comment (afiune) [inline CONTRIBUTING.md:33]: I'm not sure that this is a thing anymore... 🤔

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# This gemfile provides additional gems for testing and releasing this cookbook
+# It is meant to be installed on top of ChefDK / Chef Workstation which provide the majority
+# of the necessary gems for testing this cookbook
+#
+# Run 'chef exec bundle install' to install these dependencies
+
+source 'https://rubygems.org'
```
Comment (afiune) [inline Gemfile:7]: Do we install any gem as part of the deployment of the Lacewok Agent?

File: TESTING.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
+# Cookbook TESTING doc
+
+This document describes the process for testing the chef-lacework cookbook using ChefDK.
+
+## Testing Prerequisites
+
+A working ChefDK installation set as your system's default ruby. ChefDK can be downloaded at <https://downloads.chef.io/chef-dk/>
```
Comment (afiune) [inline TESTING.md:7]: I think we want Chef Workstation here.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+Licensing is very important to open source projects. It helps ensure the software continues to be available under the terms that the author desired.
+
+Lacework uses [the Apache 2.0 license](https://github.com/chef/chef/blob/master/LICENSE) to strike a balance between open contribution and allowing you to use the software however you would like to.
```
Comment (afiune) [inline CONTRIBUTING.md:37]: maybe link to our local LICENSE file?

File: CONTRIBUTING.md (hunk 2 above)
Comment (afiune) [reply CONTRIBUTING.md:33]: Yes, this is indeed a thing, disregard! 😅

File: CONTRIBUTING.md (hunk 5 above)
Comment (scottford-lw) [reply CONTRIBUTING.md:37]: 👍

File: Gemfile (hunk 3 above)
Comment (scottford-lw) [reply Gemfile:7]: Agreed. We don't need a `Gemfile`

**Pull request merged**